payment.sql

```diff
@@ -0,0 +1,16 @@
+-- public.payment_codes definition
```

Comment:
seems you do it manually.

I would suggest you to check this CLI
https://github.com/golang-migrate/migrate

I always use it, so do `template-go`

Reply:
sure, let me update this on part-3 task